pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -26,12 +26,12 @@ social:
 = I18n.t(:hoc_overview_subtitle)
 
 .full-resource-block{style: "margin-top: 20px;"}
-  %a{href: "/dance"}
-    %img.full-resource-image{src: "/images/dance_party.gif"}
+  %a{href: "/dance-2019"}
```

Comment:
I think this link should still go to code.org/dance since that url isn't changing, you've just updated the content on it?

Reply:
Great catch!